src/mono/netcore/sample/iOS/Makefile

```diff
@@ -1,75 +1,35 @@
 MONO_CONFIG=Debug
-
-# change to x64 for simulator
 MONO_ARCH=arm64
+
 ARTIFACTS_BIN=../../../../../artifacts/bin/
 ARTIFACTS_BCL=$(ARTIFACTS_BIN)runtime/netcoreapp5.0-iOS-$(MONO_CONFIG)-$(MONO_ARCH)
 ARTIFACTS_MONO=$(ARTIFACTS_BIN)/mono/iOS.$(MONO_ARCH).$(MONO_CONFIG)
 
 DOTNET := $(shell cd ../../ && bash init-tools.sh | tail -1)
-SYSROOT := $(shell xcrun --sdk iphoneos --show-sdk-path)
-
-BCL_LIBS = \
-	System.Runtime.dll \
-	System.Runtime.Extensions.dll \
-	System.Collections.dll \
-	System.Core.dll \
-	System.Threading.dll \
-	System.Threading.Tasks.dll \
-	System.Linq.dll \
-	System.Memory.dll \
-	System.Runtime.InteropServices.dll \
-	System.Text.Encoding.Extensions.dll \
-	Microsoft.Win32.Primitives.dll \
-	System.Console.dll
 
-all: mono-libraries aot-all xcode 
+all: mono-libraries build
+
+program:
+	$(DOTNET) build Program.csproj
+
+# copy everything to bin/ and run BuildIosApp task
+build: clean program
+	cp $(ARTIFACTS_MONO)/System.Private.CoreLib.{dll,pdb} bin
+	cp $(ARTIFACTS_MONO)/libmono.a bin
+	cp $(ARTIFACTS_BCL)/* bin
+	$(DOTNET) msbuild /t:BuildIosApp /p:Configuration=$(MONO_CONFIG) \
+	/p:TargetArchitecture=$(MONO_ARCH) \
+	/p:TargetOS=iOS \
+	/p:AppDir=$(CURDIR)/bin \
+	/p:NativeMainSource=$(CURDIR)/main.m \
+	/p:GenerateOnlyXcodeProject=true \
+	/p:DevTeamProvisioning= \
+	/p:EntryPointLib=Program.dll \
+	../../../mono.proj
```

Comment:
Another usage example - "HelloiOS" sample.
Uses `BuildIosApp` target defined in mono.proj.